### DISCUSSION TITLE
"Chat Filter" plugin

### DISCUSSION THREAD

**Discussion by uheldig** [runelite/runelite#17864] [Ideas]
I'm using "**Chat Filter**" plugin, to collapse game chat messages. (which makes the game chat more manageable)
However, this plugin has a default setting to always **Censor** or **Remove** "words" / "messages", which unnecessarily blocks a lot of sentences (that don't have slurs in them). 

My suggestion is; to add an extra option to the dropdown menu under "**Filter Type**", where you can select "None" so you can collapse game chat, without filtering public sentences/words.

Hope this makes sense. I tried explaining as simple as possible :-)

**Comment by YvesW** [comment]
You don't need to filter anything to collapse game chat messages. Please post your full chat filter plugin config.

**Comment by cdfisher** [comment]
By default the chat filter doesn't block any specific words, so if certain words are causing messages to be removed, it's because you've added them to the filter yourself.

**Comment by uheldig** [comment]
Here's some screenshots of my settings and the dropdown menu, which does'nt show other options than to always have something censored.

![1234](https://github.com/runelite/runelite/assets/170339440/00d764e5-f9c5-4a40-85c9-aa4da47181f5)
![12345](https://github.com/runelite/runelite/assets/170339440/7097b1fb-6dea-44fb-862f-be34b657ca50)

**Reply by Alexsuperfly** [reply]
uncollapse the list

**Reply by uheldig** [reply]
I'm not sure I understand, the collapse part is the whole reason I use the plugin. I would still like to collapse the game messages in my chatbox.

**Comment by YvesW** [comment]
Click this
![afbeelding](https://github.com/runelite/runelite/assets/7929021/9cb63008-9b02-48ad-b1cd-14f8674df891)

**Reply by uheldig** [reply]
Ohhh... I did not even notice this one. There's nothing under "Filtered Words" but a whole lot under "Filter Regex". (more than what one screenshot can capture)
- I assume I could just delete everything under "Regex" and it won't filter anything at all?

**Comment by YvesW** [comment]
You can just delete it. Also make sure to uninstall any "chat filter updater" plugins or something of that sort.

**Reply by uheldig** [reply]
Thank you very much!
